### PR TITLE
[ENH] yapf-ify code + add .gitignore

### PR DESCRIPTION
**Pull request by kaczmarj**
This PR proposes:
- adding a `.style.yapf` file and updating the code according to `yapf`
- adding a `.gitignore` file

The `.style.yapf` file is an explicit listing of pep8 style. The changes that `yapf` made are reasonable, but please let me know if you don't like any of the style.

to run `yapf`, I ran this on the command line:
```
yapf -rip pydra
```

This will change files in-place and in parallel.

### CODE REVIEW

**Comment by kaczmarj** [comment]
This relates to https://github.com/nipype/pydra/pull/12#issuecomment-428570385

Adding the `.style.yapf` file will prevent most (all?) user-specific differences in yapf style.

File: pydra/_version.py [hunk 1]
```diff
@@ -1,4 +1,3 @@
-
```
Comment (effigies) [inline pydra/_version.py]: Can we exclude this from `yapf`? This is an auto-generated file.

**Comment by kaczmarj** [comment]
when merged, can these commits please be squashed?

**Comment by effigies** [comment]
Is this ready to add a flake8 check in Travis? I think we might need #14 first.

**Pull request merged**